packages/components/table/src/TableCell/TableCell.tsx

```diff
@@ -17,10 +22,20 @@ export const sortingDirections = {
   desc: 'desc',
 };
 
-export type TableCellSorting = keyof typeof sortingDirections | boolean;
+export enum TableCellSorting {
+  ascending = 'ascending',
```

Comment:
```suggestion
  Ascending = 'ascending',
  Descending = 'descending',
```
It's good practice to name enum keys with PascalCase (from the TS docs: https://www.typescriptlang.org/docs/handbook/enums.html)